src/js/tech/html5.js

```diff
@@ -260,6 +298,16 @@ class Html5 extends Tech {
         const listener = listeners[eventName];
 
         elTracks.addEventListener(eventName, listener);
+
+        if (!this.trackListeners) {
+          this.trackListeners = [];
+        }
+
+        if (!this.trackListeners[props.capitalName]) {
+          this.trackListeners[props.capitalName] = [];
+        }
+
+        this.trackListeners[props.capitalName].push({ eventName, listener });
```

Comment:
what about instead of making an array of stuff, do something like the following outside of this Object.keys().forEach:
```js
this[props.getterName + 'Listeners_'] = listeners;
```
then to remove these, outside of this method we can do something like:
```js
Object.keys(this.audioTracksListeners_).forEach((eventName) => {
  elTracks.removeEventListener(eventName, this.audioTracksListeners_[eventName]
});
Object.keys(this.videoTracksListeners_).forEach((eventName) => {
  elTracks.removeEventListener(eventName, this.videoTracksListeners_[eventName]
});
```